src/syntax/select.ne

```diff
@@ -201,6 +205,9 @@ select_for -> %kw_for (
     | kw_share {% x => track(x, {type: 'share'}) %}
     | kw_key kw_share {% x => track(x, {type: 'key share'}) %})
 
+# [ NOWAIT | SKIP LOCKED ]
+select_skip -> (%kw_nowait {% x => track(x, {type: 'nowait'}) %} | %kw_skip %kw_locked {% x => track(x, {type: 'skip locked'}) %})
```

Comment:
use `kw_nowait`, `kw_skip` and  `kw_locked` you defined in `base.ne` instead of their reserved keywords counterparts (which start with a `%`)